docs/user_directory.md

```diff
@@ -8,6 +8,6 @@ who are present in a publicly viewable room present on the server.
 The directory info is stored in various tables, which can (typically after
 DB corruption) get stale or out of sync.  If this happens, for now the
 solution to fix it is to execute the SQL here
-https://github.com/matrix-org/synapse/blob/master/synapse/storage/schema/delta/53/user_dir_populate.sql
+https://github.com/matrix-org/synapse/blob/master/synapse/storage/data_stores/main/schema/delta/53/user_dir_populate.sql
```

Comment:
I'd recommend just changing this to `the SQL [here](../synapse/storage/data_stores/main/schema/delta/53/user_dir_populate.sql)` so that we it will adjust the link dynamically based on what branch the person looking is on.